_getting-started/communicate.md

```diff
@@ -0,0 +1,320 @@
+
+```json
+PUT /students/_doc/1
+{
+  "name": "John Doe",
+  "gpa": 3.91,
+  "grad_year": 2022,
+  "address": "123 Main St."
+}
+```
+{% include copy.html %}
+
+Alternatively, you can update parts of a document by calling the Update Document API:
+
+```json
+POST /students/_update/1/
+{
+  "doc": {
+    "gpa": 3.91,
+    "address": "123 Main St."
+  }
+}
+```
+{% include copy-curl.html %}
+
+For more information about partial document updates, see [Update Document API]({{site.url}}{{site.baseurl}}/api-reference/document-apis/update-document/).
+
+## Deleting documents
+
+To delete the document, send a delete request and provide the document ID:
+
+```json
+DELETE /students/_doc/1
+```
+{% include copy-curl.html %}
+
+## Deleting the index
+
+To delete the index, send the following request:
+
+```json
+DELETE /students
+```
+{% include copy-curl.html %}
+
+## Index settings and mappings
+
+OpenSearch indexes are configured with mappings and settings:
+
+- A _mapping_ is the collection of fields and the types of those fields. For more information, see [Mappings and field types]({{site.url}}{{site.baseurl}}/field-types/).
+- _Settings_ include index data like the index name, creation date, and number of shards. For more information, see [Configuring OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index/).
+
+You can specify the settings and mappings in one request. For example, the following request specifies the number of shards for the index and maps the `name` field to `text` and the `grad_year` field to `date`:
+
+```json
+PUT /students
+{
+  "settings": {
+    "index.number_of_shards": 1
+  }, 
+  "mappings": {
+    "properties": {
+      "name": {
+        "type": "text"
+      },
+      "grad_year": {
+        "type": "date"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Now you can index the same document as you indexed in the previous section:
+
+```json
+PUT /students/_doc/1
+{
+  "name": "John Doe",
+  "gpa": 3.89,
+  "grad_year": 2022
+}
+```
+{% include copy-curl.html %}
+
+To view the mappings for the index fields, send the following request:
+
+```json
+GET /students/_mapping
+```
+{% include copy-curl.html %}
+
+OpenSearch mapped the `name` and `grad_year` fields according to the types you specified and inferred the field types for the other fields:
+
+```json
+{
+  "students": {
+    "mappings": {
+      "properties": {
+        "gpa": {
+          "type": "float"
+        },
+        "grad_year": {
+          "type": "date"
+        },
+        "name": {
+          "type": "text"
+        }
+      }
+    }
+  }
+}
+```
+
+You cannot change the mappings once the index is created. 
```

Comment:
There are some mapping changes that are allowed. For example, new fields can be added. I believe you can change the search analyzer associated with a field.

Maybe "You cannot change the type of a field once it is created" ?

Reply:
Changed to your suggestion. Also added "Changing a field type requires deleting the index and recreating it with the new mappings."